internal/cmd/stack/environment.go

```diff
@@ -73,6 +119,89 @@ func setVar(cliCtx *cli.Context) error {
 	return nil
 }
 
+func (e *listEnvCommand) listEnv(cliCtx *cli.Context) error {
+	outputFormat, err := cmd.GetOutputFormat(cliCtx)
+	if err != nil {
+		return err
+	}
+
+	stackID := cliCtx.String(flagStackID.Name)
+
+	var query listEnvQuery
+	variables := map[string]interface{}{
+		"stack": graphql.ID(stackID),
+	}
+
+	if err := authenticated.Client.Query(cliCtx.Context, &query, variables); err != nil {
+		return err
+	}
+
+	var outputs []listEnvElementOutput
+	for _, attachedContext := range query.Stack.AttachedContexts {
```

Comment:
I think we should remove this - we end up duplicating the variables because they're contained in both `query.Stack.AttachedContexts` and `query.Stack.RuntimeConfig`. For example this is what I see when I run it:

![image](https://user-images.githubusercontent.com/1884632/184879560-2520409e-2983-415e-99ae-d3b031751d5f.png)

Reply:
Okay, I've removed this part as mentioned in a comment above.